src/ldp.rs

```diff
@@ -184,6 +184,12 @@ pub fn now_ms() -> DateTime<Utc> {
 pub trait LinkedDataDocument {
     fn get_contexts(&self) -> Result<Option<String>, Error>;
     fn to_value(&self) -> Result<Value, Error>;
+    fn get_default_proof_purpose(&self) -> Option<ProofPurpose> {
+        None
```

Comment:
Are those defaults for ZCAPs? Should it be implemented directly for ZCAPs directly, to make it obvious that it needs to be implemented for other LD docs?

Reply:
Added initial implementation of get_issuer in 1d9e838eca2ed2fd30804d13a7e53a7c12710aa0.
For ZCAP-LD invocation and delegation documents, there isn't the same concept of "issuer" as in VCs and VPs. So the code added in this PR is not used in ZCAP-LD. I started changing the `get_issuer` function to `get_verification_methods`, to generalize it for VC/VP and ZCAP-LD, but it's too much for this PR currently.